homeassistant/components/sharkiq/vacuum.py

```diff
@@ -101,15 +117,6 @@ def clean_spot(self, **kwargs: Any) -> None:
         """Clean a spot. Not yet implemented."""
         raise NotImplementedError
 
-    def send_command(
-        self,
-        command: str,
-        params: dict[str, Any] | list[Any] | None = None,
-        **kwargs: Any,
-    ) -> None:
-        """Send a command to the vacuum. Not yet implemented."""
-        raise NotImplementedError
-
```

Comment:
Removing this makes sense, but it's not related to adding the room target service as I understand it? Please move this change to a separate PR.